README.md

```diff
@@ -75,9 +76,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1. A stylesheet
+2. In the '<head></head>' of the html page within a  '<style></style>' tag.
+3. Within the body as a scoped attribute and only intended for just the subtree of the parent element as opposed to the whole document.
 ```
```

Comment:
:+1:  As you have listed in the order of least specific to most specific. Inline styles are the most specific.